src/Uno.UI.Wasm/tsBindings/WindowManagerRegisterUIElementParams.ts

```diff
@@ -0,0 +1,61 @@
+/* TSBindingsGenerator Generated code -- this code is regenerated on each build */
+class WindowManagerRegisterUIElementParams
+{
+	/* Pack=4 */
+	TypeName : string;
+	IsFrameworkElement : boolean;
+	Classes_Length : number;
+	Classes : Array<string>;
+	public static unmarshal(pData:number) : WindowManagerRegisterUIElementParams
+	{
+		let ret = new WindowManagerRegisterUIElementParams();
+		
+		{
+			var ptr = Module.getValue(pData + 0, "*");
+			if(ptr !== 0)
+			{
+				ret.TypeName = String(Module.UTF8ToString(ptr));
+			}
+			else
+			
+			{
+				ret.TypeName = null;
+			}
+		}
+		
+		{
+			ret.IsFrameworkElement = Boolean(Module.getValue(pData + 4, "i32"));
+		}
+		
+		{
+			ret.Classes_Length = Number(Module.getValue(pData + 8, "i32"));
+		}
+		
+		{
+			var pArray = Module.getValue(pData + 12, "*");
+			if(pArray !== 0)
+			{
+				ret.Classes = new Array<string>();
+				for(var i=0; i<ret.Classes_Length; i++)
+				{
+					var value = Module.getValue(pArray + i * 4, "*");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'value' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)